src/input.F

```diff
@@ -158,8 +159,10 @@ MODULE input
               bl_mynn_mixlength,bl_mynn_edmf,                                 &
               bl_mynn_edmf_mom,bl_mynn_edmf_tke,bl_mynn_mixscalars,           &
               bl_mynn_cloudmix,bl_mynn_mixqt,initflag,spp_pbl,bl_mynn_output, &
-              nutk,nvtk,nwtk,nparcelsLocal,nparcelsLocalActive,nparcelsMax,   &
-              nparcelsRest
+              nutk,nvtk,nwtk,nparcelsLocal,nparcelsLocalActive
+
+      real :: nparcelsInit,nparcelsMax
```

Comment:
We need to find a better solution here than to turn nparcelsInt and nparcelsMax into real.

Reply:
Revised as suggested.